asyncapi.sse.proxy/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:22.04
+
+ENTRYPOINT ["/usr/app/node_modules/.bin/sse-server"]
```

Comment:
You can delete this file. I added an sse-server image: https://github.com/aklivity/zilla-demos/pkgs/container/extras-sse-server